packages/jupyter-ai/src/handler.ts

```diff
@@ -219,4 +197,32 @@ export namespace AiService {
       completions_model_provider_id: modelId
     });
   }
+
+  export async function getModelParameters(
+    modelId?: string,
+    provider?: string
+  ): Promise<any> {
+    const params = new URLSearchParams();
+    if (modelId) params.append('model', modelId);
+    if (provider) params.append('provider', provider);
+    
+    const endpoint = `model-parameters${params.toString() ? `?${params.toString()}` : ''}`;
+    return await requestAPI(endpoint);
+  }
+
+  export async function saveModelParameters(
+    modelId: string,
+    parameters: Record<string, any>
+  ): Promise<any> {
```

Comment:
Would it be possible to improve type-safety and provide developers that might work on this code later with type hints by defining and using a more precise type for both `parameters` parameter/argument and for return type?

Reply:
@dlqqq